advanced/editor-command-identifiers.md

```diff
@@ -7,93 +7,465 @@ description: The complete list of exposed editor commands.
+| ForeColor            | Changes the text color of the text. The value passed in should be the color.                                                                                                                                                                                                                |
+| HiliteColor          | Changes the background color of the text. The value passed in should be the color.                                                                                                                                                                                                          |
+| FontName             | Font name to apply to the text. The value passed in should be the font family name.                                                                                                                                                                                                         |
+| FontSize             | Font size of the text. The value passed in should be a valid CSS font size.                                                                                                                                                                                                                 |
+| mceApplyTextcolor    | Applies text color or background color to the current selection. Requires an argument of either `'hilitecolor'` or `'forecolor'`, and the value of the color.                                                                                                                               |
+| mceRemoveTextcolor   | Removes the text color or background color from the current selection. Requires an argument of either `'hilitecolor'` or `'forecolor'`.                                                                                                                                                     |
+| RemoveFormat         | Removes any formats from the current selection.                                                                                                                                                                                                                                             |
+| mceBlockQuote        | Wraps the selected text blocks into a block quote.                                                                                                                                                                                                                                          |
+| FormatBlock          | Toggles the format of the selected block. The value passed in should be the block name. If no format is specified, the paragraph (<`p`>) format will be toggled. For a list of options, see: [FormatBlock and mceToggleFormat format options](#formatblockandmcetoggleformatformatoptions). |
```

Comment:
"Toggles the format of the current selection. The value passed in should be the format name." - but it's alright if we keep it as it is. It's a confusing command after all

Reply:
fixed